tests/serialization/test_dag_serialization.py

```diff
@@ -2538,7 +2538,11 @@ def test_operator_expand_deserialized_unmap():
     ser_normal = BaseSerialization.serialize(normal)
     deser_normal = BaseSerialization.deserialize(ser_normal)
     deser_normal.dag = None
-    assert deser_mapped.unmap(None) == deser_normal
+    unmapped_deser_mapped = deser_mapped.unmap(None)
+
+    assert type(unmapped_deser_mapped) is type(deser_normal)
+    assert unmapped_deser_mapped.task_id == deser_normal.task_id
+    assert unmapped_deser_mapped.executor_config == deser_normal.executor_config
```

Comment:
Similarly, probably should check the actual values. Also, why is checking `task_id` significant? I can understand we should check the operator’s content, but why two attributes?

Reply:
These are the two attributes that were set via `partial` on the mapped operator and with the initializer on the "normal" operator, thus they made sense to compare.

Reply:
So no particular reasons for the choice. I’m fine what that, this threas serves as record if people need a reference in the future.